app/views/schemeDetails.scala.html

```diff
@@ -21,7 +21,7 @@
 @import viewmodels.{AssociatedPsa, Message}
 
 @(appConfig: FrontendAppConfig, schemeName: String, openedDate: Option[String],
-admins: Option[Seq[AssociatedPsa]], srn: String, isSchemeOpen: Boolean, isSchemeLocked: Boolean)(implicit request: Request[_], messages: Messages)
+admins: Option[Seq[AssociatedPsa]], srn: String, isSchemeOpen: Boolean, schemeNotLocked: Boolean)(implicit request: Request[_], messages: Messages)
```

Comment:
I think isSchemeLocked is a better name than schemeNotLocked. Double negative, e.g. is false then not scheme not locked is not so easy to grasp

Reply:
fixed the flag name